data/CopyFile.java

```diff
@@ -0,0 +1,25 @@
+import java.io.*;
+
+public class CopyFile {
+    public static void main(String args[]) throws IOException {
+        int i;
```

Comment:
Name 'i' says me nothing about for what this variable needs. Please give for variable understandable names